packages/paste-core/components/tooltip/stories/index.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {Absolute} from '@twilio-paste/absolute';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {Text} from '@twilio-paste/text';
+import {Tooltip} from '../src';
+
+const Example: React.FC<{}> = () => {
+  return (
+    <Tooltip text="Welcome to Paste!">
+      <Button variant="primary">Open Tooltip</Button>
+    </Tooltip>
+  );
+};
+
+const IconButtonExample: React.FC<{}> = () => {
+  return (
+    <Tooltip text="Welcome to Paste!">
+      <Button variant="secondary" size="icon">
+        <InformationIcon decorative={false} title="Open Tooltip" />
+      </Button>
+    </Tooltip>
+  );
+};
+
+const CustomExample: React.FC<{}> = () => {
+  return (
+    <Box display="flex" alignItems="center">
+      <Text as="span">Tooltip should appear from the icon.</Text>
+      <Tooltip text="Welcome to Paste!">
+        <a href="https://paste.twilio.com" style={{display: 'inline-flex'}}>
+          <InformationIcon decorative={false} title="Open Tooltip" />
+        </a>
+      </Tooltip>
+    </Box>
+  );
+};
+
+const BottomExample: React.FC<{}> = () => {
+  return (
+    <Tooltip text="Welcome to Paste!" placement="bottom">
+      <Button variant="primary">Open Tooltip</Button>
+    </Tooltip>
+  );
+};
+
+const TopExample: React.FC<{}> = () => {
+  return (
+    <Absolute preset="bottom" bottom={12}>
+      <Tooltip text="Welcome to Paste!" placement="top-start">
+        <Button variant="primary">Open Tooltip</Button>
+      </Tooltip>
+    </Absolute>
+  );
+};
+
+const RightExample: React.FC<{}> = () => {
+  return (
+    <Tooltip text="Welcome to Paste!" placement="right-start">
+      <Button variant="primary">Open Tooltip</Button>
+    </Tooltip>
+  );
+};
+
+const LeftExample: React.FC<{}> = () => {
+  return (
+    <Absolute preset="right" top={12} right={12}>
+      <Tooltip text="Welcome to Paste!" placement="left-start">
```

Comment:
Since developers can't really know if the element they're wrapping in a tooltip will be on the edge of a screen, I believe the placement prop is more about customizing aesthetic.  Therefore, I think the stories might work better if you have 1 story called "Tooltip placements" with 4 triggers side-by-side showing all 4 placements in the same story, and another called "Automatic edge placement" which has four triggers on all 4 corners of the frame with no placement set, to show how it automatically just works.

Reply:
Also it would be cool to have a story to show automatic adjustments if the trigger is, say, on the bottom right of the frame and placement is set to "bottom" (which would ordinarily be bad in that situation)

Reply:
I've made these story adjusments.